Danger-Swift/Sources/WeTransferPRLinter/WeTransferPRLinter.swift

```diff
@@ -1,17 +1,16 @@
-// danger:disable unowned_self
```

Comment:
Things like this we need to move to after the imports, so it doesn't get removed.